Makefile

```diff
@@ -62,6 +62,9 @@ LINT_DIRS := $(DIRS) $(foreach dir,$(REC_DIRS),$(dir)/...)
 all: clean fmt check build
 
 build:
+	$(GO) build -i $(GOFLAGS) -tags '$(TAGS)' -ldflags '$(LDFLAGS)' -o '$(BINDIR)/$(BINARIES)'
```

Comment:
Is there a specific reason to use the CGO_ENABLED build as the default?

Reply:
```bash
$ make build-nocgo 
CGO_ENABLED=0 go build -i  -tags '' -ldflags '-w -s -X github.com/rancher/k3d/version.Version=v1.4.0-11-g407b153 -X github.com/rancher/k3d/version.K3sVersion=v1.17.0-k3s.1' -o '/home/alvaro/Development/k3d/bin/k3d'
go build net: open /usr/lib/go-1.13/pkg/linux_amd64/net.a: permission denied
Makefile:68: recipe for target 'build-nocgo' failed
make: *** [build-nocgo] Error 1
```

Reply:
This seems to be related to the `-i` flag.
After reading through [this issue](https://github.com/golang/go/issues/24674), I think we should rather consider removing the `-i` than switching between CGO and non-CGO (also see https://github.com/golang/go/issues/24674#issuecomment-379920545).

Reply:
Actually, according to comments on [this isse](https://github.com/golang/go/issues/25138), removing the `-i` flag in projects which use Go >= 1.10 is the better option. 
According to [the last comment there](https://github.com/golang/go/issues/25138#issuecomment-490621200), they're even going to disable installing stdlib dependencies if `-i` is set in a future release of Go.

Reply:
Cool. Fixed.